### PR TITLE
English translations

### PR DESCRIPTION
**Pull request by cbeauchesne**
I checked all messages, seems good to me. @brunobesson, you may want to double check ?

### CODE REVIEW

File: src/translations/dist/en.json [hunk 1]
```diff
@@ -264,11 +363,7 @@
     "crampons_spring": {
       "glacier_gear_types": "crampons_spring"
     },
-    "Create a new article": "Create a new article",
```
Comment (brunobesson) [inline src/translations/dist/en.json]: Why have these been removed? They still exist in the code.

File: src/translations/dist/en.json [hunk 2]
```diff
@@ -637,21 +801,30 @@
...
+    "loading": "loading in progress",
     "local_product": {
       "waypoint_types": "local product"
     },
     "locality": {
       "waypoint_types": "locality"
     },
     "Location / elevation / orientation": "Location / elevation / orientation",
-    "Log in to post the first comment": "Log in to post the first comment",
+    "Log in to post a comment": "Log in to post a comment",
```
Comment (brunobesson) [inline src/translations/dist/en.json]: I don't find the `Log in to post a comment` key, but I still find the `Log in to post the first comment`..

File: src/translations/dist/en.json [hunk 3]
```diff
@@ -842,21 +1062,31 @@
...
     "participants": "Participants",
+    "Participants and context": "Participants and context",
     "partly_protected": {
       "rain_proof_types": "partly protected"
     },
     "pass": {
       "waypoint_types": "pass"
     },
     "Password": "Password",
-    "pedestrian access": "pedestrian access",
```
Comment (brunobesson) [inline src/translations/dist/en.json]: should not be removed?

File: src/translations/dist/en.json (hunk 1 above)
Comment (cbeauchesne) [reply src/translations/dist/en.json]: I see `add an article`, but not `Create a new article`

File: src/translations/dist/en.json (hunk 2 above)
Comment (cbeauchesne) [reply src/translations/dist/en.json]: CommentsBox, line 47

File: src/translations/dist/en.json (hunk 3 above)
Comment (cbeauchesne) [reply src/translations/dist/en.json]: It has been replaced by a version with a context : https://github.com/c2corg/c2c_ui/pull/594/files/b685fb16f143b589e53909a5372d970e9a1e9ead#diff-14307f8f25eb7db0add8c8f57463ffb7R1081

**Pull request merged**